Modix.Bot/Modules/FunModule.cs

```diff
@@ -118,6 +121,24 @@ public FunModule(IHttpClientFactory httpClientFactory)
             }
         }
 
+        [Command("owoify")]
+        [Alias("owo")]
+        [Summary("Owoifies the given message.")]
+        public async Task OwoifyAsync([Remainder][Summary("The message to owoify.")] string message)
+        {
+            var owoMessage = message;
+
+            owoMessage = Regex.Replace(owoMessage, "(?:r|l)", "w");
+            owoMessage = Regex.Replace(owoMessage, "(?:R|L)", "W");
+            owoMessage = Regex.Replace(owoMessage, "n([aeiou])", "ny$1");
+            owoMessage = Regex.Replace(owoMessage, "N([aeiou])", "Ny$1");
+            owoMessage = Regex.Replace(owoMessage, "N([AEIOU])", "Ny$1");
+            owoMessage = Regex.Replace(owoMessage, "ove", "uv");
+            owoMessage = Regex.Replace(owoMessage, "\\!+", " " + _owoFaces[new Random().Next(0, _owoFaces.Length - 1)] + " ");
```

Comment:
The upper bound of Random.Next() is exclusive, so I think you can rewrite your Random call as `new Random().Next(0, _owoFaces.Length)` or even just `new Random().Next(_owoFaces.Length)` since the min is 0 by default